doc/nrf/images/zigbee_platform_design_nRF5340_multi.svg

```diff
@@ -0,0 +1,234 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Please remove the empty space by reducing the box height.

Reply:
Updated, please check back in a while.